circle.yml

```diff
@@ -138,11 +138,11 @@ commands:
           working_directory: ~/build
           command: ctest -E 'unittests|smoketests' -j4 --schedule-random --output-on-failure
 
-  collect_coverage_data:
-    description: "Collect coverage data"
+  collect_coverage_clang:
```

Comment:
Commit message says llvm, this here is clang, the text below is llvm. Choose one :)

Reply:
Although LLVM is formally more correct, I'm sticking with the Clang name.